pycbc/events/coinc.py

```diff
@@ -850,21 +850,20 @@ def pick_best_coinc(cls, coinc_results):
     @classmethod
     def from_cli(cls, args, num_templates, analysis_chunk, ifos):
         from . import stat
+
         # Allow None inputs
-        if args.statistic_files is None:
-            args.statistic_files = []
-        if args.statistic_keywords is None:
-            args.statistic_keywords = []
+        stat_files = args.statistic_files or []
```

Comment:
I haven't quite seen what is the point of the change here. If no input is given `stat_files` will be None, is this OK?

Reply:
Two changes happened here:
1. I stored the result of the operation into a local variable (`stat_files`) instead of altering `args`. This was necessary because altering `args` was messing with downstream code which also read command-line arguments.
2. I switched to a more compact implementation, using `or` instead of an `if` statement. This is not completely equivalent in general, but it is for the practical purpose of this piece of code (i.e. making sure that `stat_files` is an empty list instead of `None`).

To answer your question, if no input is given then `stat_files` will be an empty list.